ui/css/utilities/colors.scss

```diff
@@ -6,7 +6,7 @@
   --kovan: #9064ff;
   --rinkeby: #f6c343;
   --goerli: #3099f2;
-  --localhost: #29b6af;
+  --localhost: #bbc0c5;
   --flask-purple: #8b45b6;
 
   // DO NOT CHANGE
```

Comment:
Local host is the same color as mainnet but in network settings its grey. Now it matches a bit better. Our network display need an overhaul any way.

<img width="954" alt="Screen Shot 2022-05-19 at 4 03 48 PM" src="https://user-images.githubusercontent.com/8112138/169418318-59abc2fb-bc05-464c-a183-d1b8e5acb876.png">